decidim-dev/lib/decidim/dev/test/rspec_support/bullet.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "bullet"
+
+RSpec.configure do |config|
+  break unless defined?(Bullet)
+
+  if defined?(Bullet::ActiveJob)
```

Comment:
This is ugly, we should inherit all the jobs from `Decidim::ApplicationJob`

Reply:
Maybe create another issue about that?

Reply:
done in #11462